about.html

```diff
@@ -85,7 +86,7 @@ <h4>
               a forward-moving IT firm with a track record of excellence in the field of information technology.
             </p>
           </div>
-          <div class="about-logo-img"><img src="./images/Logo.png">EPALTechs<a alt="Epal Logo" style="width:60px;"></a>
+          <div class="about-logo-img"><img src="./images/Logo.png" alt="Epal Logo">EPALTechs
```

Comment:
- [ ] As you can see in the screenshots below, in the mobile version of the original design those two images should be in `flex-direction: column` instead of `row`, so kindly try to change your design to be similar to the original one, as respecting UI design is one of the most important rules that every Frontend developer should respect ^_^! 

Original design            |  Current  design
:-------------------------:|:-------------------------:
![](https://user-images.githubusercontent.com/80895497/163586874-d002cbae-6cbc-4022-b45b-6ace2c5afbea.PNG)  |  ![](https://user-images.githubusercontent.com/80895497/163586888-bc345794-6fa4-4d97-a988-66fed0de084e.PNG)